3-reveal/migrations/2-transaction_tables/deploy/plans.psql

```diff
@@ -18,6 +18,7 @@ CREATE TABLE IF NOT EXISTS plans (
     date DATE NULL,
     effective_period_start DATE NULL,
     effective_period_end DATE NULL,
+    fi_opensrpeventid VARCHAR NULL,
```

Comment:
This needs to be added as a separate database migrations ([here is an example](https://github.com/OpenSRP/opensrp-reveal-datawarehouse/blob/master/3-reveal/migrations/2-transaction_tables/deploy/more_event_fields.psql)).  We cannot modify it directly because we want it to run as a DB migration that can update already existing tables.